config/src/main/java/org/axonframework/config/EventProcessingModule.java

```diff
@@ -771,39 +779,45 @@ private TrackingEventProcessorConfiguration trackingEventProcessorConfig(String
         return tepConfigs.getOrDefault(name, defaultTrackingEventProcessorConfiguration).get();
     }
 
-    private SubscribingEventProcessor subscribingEventProcessor(String name,
-                                                                EventHandlerInvoker eventHandlerInvoker,
-                                                                SubscribableMessageSource<? extends EventMessage<?>> messageSource) {
+    /**
+     * @param name
+     * @param eventHandlerInvoker
+     * @param messageSource
+     * @return
+     */
+    protected EventProcessor subscribingEventProcessor(String name,
```

Comment:
Again to be sure about the intent. We made these methods public so that we could construct a tenant-specific `EventProcessingModule` that overrides these, right?